pkg/controller/controller.go

```diff
@@ -44,7 +44,7 @@ func AddToManager(m manager.Manager, cfg *rest.Config, syncid *types.NamespacedN
 
 	// If remote subscription pod (appmgr) is running in hub, don't add helmrelease controller to the manager,
 	// As there has been a helmrelease controller running in standalone subscription pod
-	if !utils.IsHub(m.GetConfig()) || standalone {
+	if !utils.IsHub(m.GetConfig()) || ops.Standalone {
```

Comment:
@ianzhang366 @xiangjingli do you think we should change this line to
```
if ops.Standalone || !utils.IsHub(m.GetConfig()) {
```
I am not sure if the golang compiler already does this optimization automatically but its better if we just do it ourselves to make sure. I can make the change to this branch if you guys agree.

Reply:
good catch, but this PR is planning to be merged after the 2.2 release, if that timeline is ok for you, please make the change